stores/settings.go

```diff
@@ -0,0 +1,281 @@
+package stores
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+
+	"go.sia.tech/renterd/api"
+	sql "go.sia.tech/renterd/stores/sql"
+	"go.uber.org/zap"
+)
+
+const (
+	SettingGouging = "gouging"
+	SettingPinned  = "pinned"
```

Comment:
This naming is a bit odd to me. These are the price pinning settings to I think `SettingsPricePinning` and `pricepinning` makes more sense.

Reply:
Ok then let's keep it to make things easier in the frontend.